ratis-metrics/src/main/java/org/apache/ratis/metrics/RatisMetrics.java

```diff
@@ -31,9 +31,16 @@
   protected RatisMetricRegistry registry;
 
   protected static RatisMetricRegistry create(MetricRegistryInfo info) {
+    return create(info, true);
+  }
+
+  protected static RatisMetricRegistry create(MetricRegistryInfo info,
+      boolean shouldDebugLog) {
     Optional<RatisMetricRegistry> metricRegistry = MetricRegistries.global().get(info);
     return metricRegistry.orElseGet(() -> {
-      LOG.info("Creating Metrics Registry : {}", info.getName());
+      if (shouldDebugLog) {
+        LOG.debug("Creating Metrics Registry : {}", info.getName());
```

Comment:
We have INFO log in the same file:

```
   LOG.info("Unregistering Metrics Registry : {}", info.getName());
```

We can also modify the debug level here...

Reply:
Done.